aspnetcore/data/ef-mvc/intro.md

```diff
@@ -383,16 +388,16 @@ Advance to the next tutorial to learn how to perform basic CRUD (create, read, u
 
 [!INCLUDE [RP better than MVC](~/includes/RP-EF/rp-over-mvc.md)]
 
-The Contoso University sample web application demonstrates how to create ASP.NET Core 2.2 MVC web applications using Entity Framework (EF) Core 2.2 and Visual Studio 2017 or 2019.
+The Contoso University sample web application demonstrates how to create ASP.NET Core 6.0 MVC web applications using Entity Framework (EF) Core 6.0.9 and Visual Studio 2022 or 2022.
 
 This tutorial has not been updated for ASP.NET Core 3.1. It has been updated for [ASP.NET Core 5.0](xref:data/ef-mvc/intro?view=aspnetcore-5.0&preserve-view=true).
 
 The sample application is a web site for a fictional Contoso University. It includes functionality such as student admission, course creation, and instructor assignments. This is the first in a series of tutorials that explain how to build the Contoso University sample application from scratch.
 
 ## Prerequisites
 
-* [.NET Core SDK 2.2](https://dotnet.microsoft.com/download)
-* [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=learn.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019) with the following workloads:
+* [.NET Core SDK 6.0](https://dotnet.microsoft.com/download)
+* [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=learn.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2022) with the following workloads:
```

Comment:
This is in the moniker section for earlier than 3.1, so it can't be updated to .NET 6 / VS2022.

```suggestion
* [.NET Core SDK 2.2](https://dotnet.microsoft.com/download)
* [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=learn.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019) with the following workloads:
```